wallet-new/src/Cardano/Wallet/API/V1/Transactions.hs

```diff
@@ -28,4 +28,12 @@ type API = Tags '["Transactions"] :>
                         :> Summary "Estimate the fees which would originate from the payment."
                         :> ReqBody '[ValidJSON] Payment
                         :> Post '[ValidJSON] (WalletResponse EstimatedFees)
+    :<|> "external-transactions" :> "unsigned"
+                        :> Summary "Creates a new, raw, unsigned transaction (it will be signed externally)."
+                        :> ReqBody '[ValidJSON] Payment
+                        :> Post '[ValidJSON] (WalletResponse Transaction)
+    :<|> "external-transactions"
+                        :> Summary "Publish an externally-signed transaction."
+                        :> ReqBody '[ValidJSON] SignedTransaction
+                        :> Post '[ValidJSON] (WalletResponse Transaction)
```

Comment:
Minor API quibble, but it seems like a cleaner way to do this is `/api/v1/transactions/external/` ?